2022-round-2/jeftersantiago/nx_tutorial_script.py

```diff
@@ -0,0 +1,36 @@
+import networkx as nx
+import matplotlib.pyplot as plt
+
+G = nx.DiGraph()
+
+# Adds nodes of type int
+G.add_node(1)
+G.add_node(2)
+
+# Adds nodes of type str
+G.add_node('São Paulo')
+G.add_node('New York')
+
+# Add nodes of tuples
+G.add_nodes_from([(3, 'Beijing'), (4, 'Moscow')])
+
+# Add edges between nodes
+
+G.add_edge(1, 2)
+G.add_edge(2, 1)
+G.add_edge('São Paulo', 2)
+G.add_edge(2, (3, 'Beijing'))
+G.add_edge((3, 'Beijing'), 'New York')
+G.add_edge((4, 'Moscow'), 'São Paulo')
+G.add_edge((4, 'Moscow'), 'New York')
+
+# Calculates the shortest distances between all pairs
+path = dict(nx.all_pairs_dijkstra_path_length(G))
```

Comment:
Any particular reason you aren't using `all_pairs_shortest_path_length` here?

Reply:
No particular reason. It makes more sense to use `all_pairs_shortest_path_length` for a simple graph like this, instead of the function used. It's okay if i change it?
Thanks for the review :)